### PR TITLE
Shruti Assignment22-35 mints

### PR DESCRIPTION
**Pull request by ShrutiDJun20**
For String:
image

For Number:
image

### CODE REVIEW

**Review by PalakSJun20 — APPROVED** [review]
Code Looks good to me

File: src/shrutiDJun20Assignments/PalindromeString1.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+		if (input.equalsIgnoreCase(rev))
+			System.out.println("Given string '" + input + "' is palindrome.");
+		else
+			System.out.println("Given string '" + input
+					+ "' is not palindrome.");
+
+	}
+	public static void main(String[] args) {
+		PalindromeString1 palindromeString1 = new PalindromeString1();
+		palindromeString1.palindromeStringCheck("Radar");
```
Comment (KrishnaKTechnocredits) [inline src/shrutiDJun20Assignments/PalindromeString1.java]: @ShrutiDJun20 @PalakSJun20 "Radar" is not a palindrome string. "radar" is . It is case sensitive

File: src/shrutiDJun20Assignments/PalindromeString1.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package shrutiDJun20Assignments;
+
+public class PalindromeString1 {
+
+	void palindromeStringCheck(String input) {
+		String rev = "";
+		for (int index = input.length()-1; index >= 0; index--) {
+			rev = rev + input.charAt(index);
+		}
+		if (input.equalsIgnoreCase(rev))
```
Comment (KrishnaKTechnocredits) [inline src/shrutiDJun20Assignments/PalindromeString1.java]: It has to be equals not equalsIgnoreCase.

Comment (ShrutiDJun20) [reply src/shrutiDJun20Assignments/PalindromeString1.java]: changes done

File: src/shrutiDJun20Assignments/PalindromeString1.java (hunk 1 above)
Comment (ShrutiDJun20) [reply src/shrutiDJun20Assignments/PalindromeString1.java]: changes done

**Pull request merged**